trestle/tasks/ocp4_cis_profile_to_oscal_catalog.py

```diff
@@ -202,14 +202,17 @@ def _parse(self, lines: List[str]) -> None:
 
     def _get_key(self, name: str) -> (int, int, int):
         """Convert name to desired sortable key."""
-        key = 0
         parts = name.split('.')
         if len(parts) == 1:
             key = (int(parts[0]), 0, 0)
         elif len(parts) == 2:
             key = (int(parts[0]), int(parts[1]), 0)
         elif len(parts) == 3:
             key = (int(parts[0]), int(parts[1]), int(parts[2]))
+        else:
+            text = f'Unexpected value: {name}'
+            logger.error(text)
```

Comment:
Recently we adopted the structure to not use `logger.error` to avoid the repetition of the error in the console, please simply raise the exception with error message and let it fail the program.

Reply:
Fixed.